app/assets/javascripts/discourse/app/components/welcome-topic-banner.js

```diff
@@ -0,0 +1,35 @@
+import { getOwner } from "discourse-common/lib/get-owner";
+import { action } from "@ember/object";
+import Component from "@ember/component";
+import Topic from "discourse/models/topic";
+import Composer from "discourse/models/composer";
+import { on } from "discourse-common/utils/decorators";
+
+export default Component.extend({
```

Comment:
Now that Glimmer components are fully supported, all new components should be using Glimmer components. The old Ember components are something we'll move away from soon.